src/components/MegadraftEditor.js

```diff
@@ -152,6 +154,23 @@ export default class MegadraftEditor extends Component {
     }
   }
 
+  getSidebarComponent = () => {
```

Comment:
That's a good point but I believe it won't be a problem, we can control the props from the `render` method.